lib/containers/runtime.pm

```diff
@@ -131,18 +222,178 @@ sub remove_image {
     $self->_rt_assert_script_run("rmi -f $image_name");
 }
 
+=head2 remove_container
+
+Remove a container from the pool.
+
+=cut
+sub remove_container {
+    my ($self, $container_name) = @_;
+    $self->_rt_assert_script_run("rm -f $container_name");
+}
+
+=head2 check_image_in_host_registry
+
+Returns true if host contains C<img> or false.
+
+=cut
+sub check_image_in_host_registry {
+    my ($self, $img) = @_;
+    my @lregistry = $self->enum_images();
+    $img =~ @lregistry;
+}
+
+=head2 setup_registry
```

Comment:
i was thinking based on another comment of you to just replace the setup_registry with `allow_selected_insecure_registries` or even better `configure_insecure_registries`. good?

Reply:
That's also ok. But in case we want to add some more configuration later on not related to registry, we can just have a generic name. `configure` or `setup` would work. But it's just my opinion, we can always change that later.

Reply:
I agree with both options too :smile:

Reply:
*this change is ready for review*